torch/cuda/__init__.py

```diff
@@ -584,16 +585,17 @@ def get_sync_debug_mode() -> int:
 from ..storage import _StorageBase
 
 
-if not hasattr(torch._C, 'CudaDoubleStorageBase'):
+if not hasattr(torch._C, 'CudaByteStorageBase'):
     # Define dummy base classes
     for t in ['Double', 'Float', 'Long', 'Int', 'Short', 'Char', 'Byte', 'Half', 'Bool', 'BFloat16',
               'ComplexDouble', 'ComplexFloat']:
-        storage_name = 'Cuda{0}StorageBase'.format(t)
         tensor_name = 'Cuda{0}TensorBase'.format(t)
 
-        torch._C.__dict__[storage_name] = _dummy_type(storage_name)
         torch._C.__dict__[tensor_name] = _dummy_type(tensor_name)
 
+    storage_name = 'CudaByteStorageBase'
+    torch._C.__dict__[storage_name] = _dummy_type(storage_name)
```

Comment:
Hey @kurtamohler. I'm going over public API changes for the 1.11 release, and it looks like `torch.CudaByteStorageBase` was added some time between 1.10 and now. I think that's coming from this PR - does that look intentional? There aren't any public docs for it.

Reply:
Same thing actually goes for TypedStorage (added here: https://github.com/pytorch/pytorch/pull/62030/files#diff-9189d159e01ea5173c234ead87f40f4a99cb1aa01264a3259b99a162fee4248cR624). should we just `del TypedStorage` at the end of this file?

Reply:
`torch.CudaByteStorageBase` isn't added in this PR, and it was in 1.10:

```python
>>> torch.__version__
'1.10.2+cu113'
>>> torch.CudaByteStorageBase
<class 'torch._C.CudaByteStorageBase'>
```

`TypedStorage` and `UntypedStorage` were added in this PR, and they have not been documented yet. #66970 will add documentation for `TypedStorage`. I was planning on documenting `UntypedStorage` after #66970

Reply:
> torch.CudaByteStorageBase isn't added in this PR, and it was in 1.10:

Whoops you're right (it might because I wasn't building with cuda when I checked)

> TypedStorage and UntypedStorage were added in this PR, and they have not been documented yet. https://github.com/pytorch/pytorch/pull/66970 will add documentation for TypedStorage. I was planning on documenting UntypedStorage after https://github.com/pytorch/pytorch/pull/66970

For the 1.11 release, we'll either want them to be documented, or to remove them from python / mark them private (cc @albanD).

How heavily used are they currently? If docs aren't likely to be ready in the next week or two, then what do you think of marking them with leading underscores for now? (`_TypedStorage`, `_UntypedStorage`)

Reply:
Yeah, adding underscores is probably a good idea. Users shouldn't need to interact with them directly until #66970 lands. I can submit a PR to change them if you'd like

cc @ezyang, just in case he sees any issue with this

Reply:
That sounds good to me. Thanks!

Reply:
Hey @ezyang, it looks like landing that PR will require internal changes, since there are internal uses of `torch.[Un]TypedStorage`. I'm wondering if you have any thoughts on marking those API's private for the 1.11 release?

Reply:
Sounds good to me.